src/ctl/src/cmd_impl/hummock/sst_dump.rs

```diff
@@ -320,7 +320,7 @@ fn print_kv_pairs(
         let full_val = block_iter.value();
         let humm_val = HummockValue::from_slice(full_val)?;
 
-        let epoch = Epoch::from(full_key.epoch);
+        let epoch = Epoch::from(full_key.epoch_with_gap.get_epoch());
```

Comment:
If current `get_epoch` is to return an epoch with gap, here we may take its pure epoch instead of the epoch with gap?

Reply:
Or we print the gap separately later.

Reply:
Shall we provide a `epoch` method for full key to directly return its inner pure epoch?

Reply:
> Shall we provide a epoch method for full key to directly return its inner pure epoch?

Agree.